airbyte-webapp/src/components/ui/Text/Text.tsx

```diff
@@ -4,13 +4,15 @@ import React from "react";
 import styles from "./text.module.scss";
 
 type TextSize = "xs" | "sm" | "md" | "lg";
+type TextColor = "darkBlue" | "grey";
```

Comment:
ℹ️ We have more and more cases when we need to have `grey/default` text color, so I've extended the `<Text />` component to support color